components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/widget/Option.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.dashboards.core.bean.widget;
+
+import java.util.List;
+
+/**
+ * Configuration bean class for options in widget conf file.
+ */
+public class Option {
+
+    enum Types {
+        TypeText, TypeEnum, TypeBoolean
```

Comment:
As mentioned in the code reviews, lets change these enums to `TEXT`, `ENUM`, `BOOLEAN`

Reply:
changed with 910980c413d9aea940c47a751f949593c1fc5ec0